src/scroll-timeline-base.js

```diff
@@ -476,6 +477,61 @@ export class ScrollTimeline {
   static isValidAxis(axis) {
     return ["block", "inline", "x", "y"].includes(axis);
   }
+
+  // Calculate the fractional offset of a range value relative to the full range.
+  relativePosition(value, details) {
```

Comment:
It looks like there are a few previous existing offenders that slipped in, but the goal of the polyfill is to present only the exact interface of ScrollTimeline / ViewTimeline. Any methods / helpers which aren't part of the public javascript interface can be free functions that accept the timeline as a parameter, e.g. see the [updateInternal function](https://github.com/flackr/scroll-timeline/blob/147d187620e49ce5fb3abc9e1c3e1018755d8f02/src/scroll-timeline-base.js#L34).

Reply:
Got it.

Reply:
Done.